CMakeLists.txt

```diff
@@ -162,7 +162,22 @@ if(SIMPATH)
 endif()
 
 # FairRoot only supports ROOT6, so check which version is available
-find_package2(PUBLIC ROOT  VERSION 6.10.00  REQUIRED)
+list(APPEND required_root_components RIO)
+if(BUILD_SIM)
+  list(APPEND required_root_components pythia6 pythia8)
+endif()
+list(APPEND optional_root_components opengl)
+list(APPEND optional_root_components gdml vmc)
+find_package2(PUBLIC ROOT REQUIRED
+  VERSION 6.18.00
+  COMPONENTS ${required_root_components}
+  OPTIONAL_COMPONENTS ${optional_root_components})
+Find_Program(ROOT_CINT_EXECUTABLE
+  NAMES rootcint
+  PATHS ${ROOT_BINDIR}
+  NO_DEFAULT_PATH
+  )
+include(ROOTMacros)
```

Comment:
```suggestion
```

Nothing in this module should be in use in FairRoot any more, why include it? (The file may be used by experiment repos however)

Reply:
Ah, I see, it was included in the deleted `FindROOT.cmake`

Reply:
And without this, some things don't work. Don't ask me what. I forgot about it.

So maybe experiments will need to add this `include` somewhere? Because the `find_package` wont load it any more.